pkg/repo/guild_config_admin_role/pg.go

```diff
@@ -0,0 +1,37 @@
+package guild_config_admin_role
+
+import (
+	"gorm.io/gorm"
+
+	"github.com/defipod/mochi/pkg/model"
+)
+
+type pg struct {
+	db *gorm.DB
+}
+
+func NewPG(db *gorm.DB) *pg {
+	return &pg{
+		db: db,
+	}
+}
+
+func (pg *pg) Create(config []model.GuildConfigAdminRole) error {
+	tx := pg.db.Begin()
+	for _, item := range config {
```

Comment:
create can insert batch, no need to loop all item
https://gorm.io/docs/create.html#Batch-Insert